monitoring/chart-values/prometheus-values.yaml

```diff
@@ -70,14 +70,25 @@ alertmanager:
         # text:
     # templates:
     #   - '/etc/alertmanager/config/*.tmpl'
-
+  tolerations:
+    - key: o11y
+      value: "true"
+      effect: NoSchedule 
+  nodeSelector:
+    workload: o11y
 ###kube-prometheus-stack###
 grafana:
   grafana.ini:
   enabled: true
   env:
-    GF_SERVER_ROOT_URL: "%(protocol)s://%(domain)s:/grafana"
+    GF_SERVER_ROOT_URL: "http://localhost:3000/grafana"
```

Comment:
we can make it back to `"%(protocol)s://%(domain)s:/grafana"` since we are accessing  it via LB